index.js

```diff
@@ -0,0 +1,46 @@
+import express from 'express'
+
+const app = express();
+const PORT = process.env.PORT ?? 3000;
+
+app.use(express.json());
+
+app.get('/api/greetings', (request, response) => {
+  const name = request.query.name;
+  name ?
+  response.send(`<p>Hello ${name}!</p>`)
+  : response.status(400).json({ error: 'Name undefined' })
```

Comment:
It's a pretty bad way to write conditions.
You can use instead:
```
if (name) {
  response.send(`<p>Hello ${name}!</p>`)
} else {
  response.status(ERROR_STATUS).json({ error: 'Name undefined' })
}
```

Move please MAGIC NUMBER to the separate constant from constants file